frontend/src/sass/common/uclapi.scss

```diff
@@ -0,0 +1,439 @@
+	of interacting with css is removed.
+
+	Table of contents:
+
+	- Setup
+	- Color Scheme
+	- Sizings
+	- Text
+	- Alignments
+	- Containers
+	- Buttons
+	- Transitions
+	- Cards
+	- Code Segments
+	- RC-Collapse
+  - Input Forms
+  - Auto complete forms
+  - Code View
+
+**/
+
+/** Setup **/
+
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap');
+
+$break-small: 780px;
+
+html, button, input, select, textarea {
+	font-family: 'Source Sans Pro','Roboto', sans-serif;
+}
+
+html, body {
+  margin: 0;
+  background-color: white;
+}
+
+/** Color Scheme **/
+
+$primary-color: #296da7;
+$primary-highlight-color: #3498db;
+
+$secondary-color: #313537;
+
+$accent-color: #8D6E63;
+$highlight-color: #B18453;
+
+/** Row Colours **/
+
+.splash-parallax { 
+  background-image: url("~Images/home-page/splash_screen.png");
+  box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.4); 
+  
+  /* Full height */
+  height: 100%; 
+
+  /* Create the parallax scrolling effect */
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.team-parallax { 
+  background-image: url("~Images/team/teambg.jpg");
+  box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.4); 
+  
+  /* Full height */
+  height: 100%; 
+
+  /* Create the parallax scrolling effect */
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.primary-highlight { background-color: $primary-highlight-color }
+.primary { background-color: $primary-color }
+
+.secondary { background-color: $secondary-color;}
+.warning-red { background-color: red; }
+.market-green { background-color: #03B77B; }
+.transparent { background-color: #FFFFFF00; }
+
+/** Text **/
+
+p, h1, h2, h3, h4, h5, h6 {
+	font-family: 'Source Sans Pro', 'Roboto', sans-serif;
+	color: white;
+}
+
+p, h1, h2, h3, h4, h5 {
+  margin-top: 0;
+}
+
+h5 {
+	font-weight: normal
+}
+
+h1 {
+	font-size: 42px;
+    font-weight: 700;
+    margin-bottom: 20px;
+    margin-top: 0;
+    text-shadow: 0 0 5px rgba(0, 0, 0, 0.45);
+}
+
+.justified-text {
+  text-align: justify;
+}
+
+.centered-text {
+	text-align: center;
+}
+
+a{ color: white; text-decoration-color: white; }
+a:hover { color: #BBBBBB; }
+
+.alt-text { color: $primary-color; text-decoration-color: #313537; }
+.alt-text:hover { color: $primary-highlight-color; }
+
+/** Alignments **/
+
+.full-screen {
+	height: 100%;
+}
+
+.column {
+    text-align: center;
+    overflow: hidden;
+}
+
+.center-x {
+	position: relative;
+	margin: auto;
+}
+
+.vertical-align {
+	width: 100%;
+	position: absolute;
+}
+
+.no-vertical-align {
+	width: 100%;
+	position: relative;
+}
+
+.center-y {
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+.bottom-y {
+  bottom: 50px;
+}
+
+/** Containers **/
+
+.row {
+	width: 100%;
+  	position: relative;
+	height: auto;
+	min-height: 100% !important;
+}
+
+.vertical-padding {
+  padding: 50px 0;
```

Comment:
My personal opinion is that 50px is a bit much. I think 20px will do just fine.

Reply:
<img width="643" alt="Annotation 2019-10-17 194644" src="https://user-images.githubusercontent.com/6523121/67038259-f8f6a900-f116-11e9-9351-7124ddfc0820.png">
<img width="643" alt="Annotation 2019-10-17 194611" src="https://user-images.githubusercontent.com/6523121/67038261-f98f3f80-f116-11e9-8310-ab9fe7173776.png">

Reply:
Also, the bottom margin for the h5 makes the vertical padding uneven.
<img width="311" alt="Annotation 2019-10-17 194934" src="https://user-images.githubusercontent.com/6523121/67038430-45da7f80-f117-11e9-93b4-a80b211d68e8.png">

Reply:
There should also be some horizontal padding

![image](https://user-images.githubusercontent.com/6523121/67043069-1fb9dd00-f121-11e9-9049-f1b8577851b4.png)